src/ui/utils/tokenManager.tsx

```diff
@@ -6,10 +6,20 @@ import { useHistory } from "react-router-dom";
 import { assert, defer, Deferred } from "protocol/utils";
 
 const domain = "webreplay.us.auth0.com";
-const clientId = "4FvFnJJW4XlnUyrXQF8zOLw6vNAH1MAo";
 const audience = "hasura-api";
 const tokenRefreshSecondsBeforeExpiry = 60;
 
+let clientId: string;
+const isTesting = new URL(window.location.href).searchParams.get("e2etest");
+
+if (isTesting) {
+  // Auth0 Application: Cypress E2E Testing
+  clientId = "uL3A8jxVqotF5Q6bmn5QTV46hU97MPQm";
+} else {
+  // Auth0 Application: Replay
+  clientId = "4FvFnJJW4XlnUyrXQF8zOLw6vNAH1MAo";
+}
```

Comment:
nit, but leets use a ternary and drop cypress... we'll need this without cypress as well

```
// Auth0 E2E tests require a different clientId
const clientId = isTesting ? <> : <>;
```